index.html

```diff
@@ -57,5 +59,204 @@ <h1>
+        </div>
+        <div href="#" class="see-project"><a>See Project</a></div>
+      </div>
+    </div>
+
+    <!-- Project 2 -->
+
+    <div class="work-section">
+      <div class="snapshot">
+        <img src="./images/snapshot-2.svg" alt="" />
+      </div>
+      <div class="contain">
+        <p class="snapshot-title">
+          Multi-Post <br />
+          Stories
+        </p>
+        <div class="client-info">
+          <p class="client">CANOPY</p>
+          <div class="circle"></div>
+          <p>Back End Dev</p>
+          <div class="circle"></div>
+          <p>2015</p>
+        </div>
+        <p class="project-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="used-language">
+          <div class="language">html</div>
+          <div class="language">css</div>
+          <div class="language">javascript</div>
+        </div>
+        <div href="#" class="see-project"><a>See Project</a></div>
+      </div>
+    </div>
+
+    <!-- Project 3 -->
+
+    <div class="work-section">
+      <div class="snapshot">
+        <img src="./images/snapshot-3.svg" alt="" />
+      </div>
+      <div>
+        <p class="snapshot-title">Tonic</p>
+        <div class="client-info">
+          <p class="client">CANOPY</p>
+          <div class="circle"></div>
+          <p>Back End Dev</p>
+          <div class="circle"></div>
+          <p>2015</p>
+        </div>
+        <p class="project-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="used-language">
+          <div class="language">html</div>
+          <div class="language">css</div>
+          <div class="language">javascript</div>
+        </div>
+        <div href="#" class="see-project"><a>See Project</a></div>
+      </div>
+    </div>
+
+    <!-- Project 4 -->
+
+    <div class="work-section">
+      <div class="snapshot">
+        <img src="./images/snapshot-4.svg" alt="" />
+      </div>
+      <div>
+        <p class="snapshot-title">
+          Multi-Post <br />
+          Stories
+        </p>
+        <div class="client-info">
+          <p class="client">CANOPY</p>
+          <div class="circle"></div>
+          <p>Back End Dev</p>
+          <div class="circle"></div>
+          <p>2015</p>
+        </div>
+        <p class="project-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="used-language">
+          <div class="language">html</div>
+          <div class="language">css</div>
+          <div class="language">javascript</div>
+        </div>
+        <div href="#" class="see-project"><a>See Project</a></div>
+      </div>
+    </div>
+
+    <!-- Work Section Ends -->
+
+    <!-- About Section Starts -->
+
+    <div class="about-section">
+      <h1>
```

Comment:
- Just as i said earlier about the use of `<section>`, It will be better for line 189 to be in a section tag instead. So you can refactor to 👉🏽 `<section class="about-section">`. 🌸

Reply:
Thanks. Done!